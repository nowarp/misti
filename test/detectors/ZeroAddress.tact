contract SampleContract {
    init() {
        newAddress(1, 0x000000000000000000000000000000000000000000000000);
    }
}
