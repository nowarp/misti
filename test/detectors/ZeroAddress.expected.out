[LOW] ZeroAddress: Using zero address
test/detectors/ZeroAddress.tact:3:23:
  2 |     init() {
> 3 |         newAddress(1, 0x000000000000000000000000000000000000000000000000);
                            ^
  4 |     }
Help: Consider changing code to avoid using it. For example, you could pass the address during the deployment.
See: https://nowarp.io/tools/misti/docs/detectors/ZeroAddress